publish-frontend/src/pages/posts/preview/[postId].js

```diff
@@ -83,8 +82,8 @@ export default function PreviewArticlePage({ post, unsavedPostContent }) {
         description: `This is just a preview of the formatting of the content for readability. The page may look different when published on the publication.`,
         isClosable: true,
         status: 'info',
-        position: 'bottom-left',
-        duration: null
+        position: 'bottom-right',
+        duration: 5000
```

Comment:
```suggestion
        duration: null
```
Having them auto-dismiss isn't great from an a11y perspective, since it's hard to predict how long it will take someone to read them.